src/track/beatutils.cpp

```diff
@@ -30,6 +30,7 @@ const double kHistogramDecimalScale = pow(10.0, kHistogramDecimalPlaces);
 const double kBpmFilterTolerance = 1.0;
 const double k23Epsilon = 0.1;
 const double kRoundFactor = 0.05;
+const double kMinMachineTempoDeviaton = 0.0075;
```

Comment:
How did you determine this constant? Is it an arbitrary guess? Did you guess and check with a dataset until you found an optimal value?

Reply:
Tried on a few (~10) electronic tracks that I had handy, they were all around 0.006 or 0.005 something and also tried on some of the GTZAN tracks of different genres and they were all greater than 0.01.  So I just guessed that this would be a good value as a starting point, probably not optimal.
Hopefully, @poelzi can test in with his tracks and tweak it to an optimal value